src/vaults/BalanceSheet.sol

```diff
@@ -124,21 +120,10 @@ contract BalanceSheet is Auth, Recoverable, IBalanceSheet, IBalanceSheetGatewayH
             tokenId,
             receiver,
             amount,
-            pricePoolPerAsset,
-            m
+            pricePoolPerAsset
         );
     }
 
-    /// @inheritdoc IBalanceSheet
-    function updateValue(PoolId poolId, ShareClassId scId, address asset, uint256 tokenId, D18 pricePoolPerAsset)
-        external
-        auth
-    {
-        uint128 assetId = poolManager.assetToId(asset, tokenId);
-        sender.sendUpdateHoldingValue(poolId, scId, AssetId.wrap(assetId), pricePoolPerAsset);
-        emit UpdateValue(poolId, scId, asset, tokenId, pricePoolPerAsset, uint64(block.timestamp));
-    }
```

Comment:
Talked with Frederik about removing this and avoid concurrent issues with the counterpart in CP side.